cosmo_tester/framework/test_hosts.py

```diff
@@ -817,24 +822,40 @@ class CloudifyDistributed_MessageQueue(_CloudifyMessageQueueOnly):
     image_name = get_latest_manager_image_name()
 
 
+class CloudifyDistributed5_0Manager(CloudifyDistributed_Manager):
```

Comment:
maybe more like 5_0_0_Manager

Reply:
Just being consistent with the rest of the classes in the file

Reply:
it's a mistake because there will be versions in between 5.1 and 5.0

Reply:
Then we should update all the version in a different dedicated-for-the-event commit

Reply:
fixed